tests/test.js

```diff
@@ -438,4 +438,173 @@ test('NbsDecoder.getPackets() returns the last packet of each type when given a
   });
 });
 
+const multiTypeNextTimestampArray = [
+  { type: pongType, subtype: 0 },
+  { type: pingType, subtype: 0 },
+  { type: pangType, subtype: 100 },
+  { type: pangType, subtype: 200 },
+];
+
+test('NbsDecoder.nextTimestamp() outputs the exact timestamp as input, given steps = 0', () => {
+  const [start, end] = decoder.getTimestampRange({ type: pingType, subtype: 0 });
+  // const new_timestamp = decoder.nextTimestamp(start, { type: pingType, subtype: 0 }, 0);
+  const new_timestamp = decoder.nextTimestamp(start, { type: pingType, subtype: 0 }, 0);
```

Comment:
This test isn't really testing the desired behaviour.
Instead of `nextTimestamp` from start, do `nextTimestamp` from start + 1 second (should be the same Ping message).
That should return the start timestamp rather than start + 1

Reply:
Refactored to start + 1.